UKHomeOffice/nmsw-ui
# NMSW-1393 Landing page wording has crew instead of passengers above FAL6 template

# Ticket

NMSW-1393

----

## AC

Navigate to NMSW external landing page

Expected behaviour:
- Content above FAL6 template should say all Passengers

Actual behaviour:
- Content above FAL6 template has all Crew 

----

## To test

- run app
- navigate to NMSW external landing page
- > You should see content above FAL 6 download to say 'passengers'
- > You should see content above FAL 5 download to have the word 'other' removed


----

## Notes
